makefile.shade

```diff
@@ -45,23 +45,41 @@ use-standard-lifecycle
           return;
       }
 
-      foreach (var package in Directory.EnumerateDirectories(PACKAGE_DROP_SHARE))
+      var latestPackageFolders = Directory.GetDirectories(PACKAGE_DROP_SHARE);
+      var coherentPackageNames = Directory.GetFiles(BUILD_DIR, "*.nupkg").Select(p => Path.GetFileNameWithoutExtension(p));
```

Comment:
Maybe we could move this into SanityCheck - this doesn't have access to SanityCheck's excludes list so you might end up getting extra stray packages if a repo gets ignored but the directory stays behind (for instance renaming). This can be done as a follow up item since it's not super urgent.